content/en/kb/butterfly-network/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "How to join the Butterfly network."
+description: "A guide for how you can join the Butterfly network"
+date: 2022-03-17T12:00:35+01:00
+lastmod: 2021-11-16T12:00:35+01:00
+draft: false
+menu:
+  kb:
+    parent: "browse"
+toc: false
+pinned: false
+types: ["solution"]
+areas: ["Lotus Node", "Lotus Miner"]
+---
+
+## Problem
+
+How to join the `Butterfly` network.
```

Comment:
We need to add full description here about butterfly net.
purpose
Use case
How to check notification for reset
Possible reset reasons
Why should you join it

Reply:
I added description about the network/purpose/reset notifications/subscription.

 Not sure if we should add `possible reset reasons` and `Why should you join it` just yet, as it´s still mostly an internal development network that happens to be open to everybody. If we change it to a more open developer network at some point, I think we should add those points though! Wdyt?

Reply:
Fair enough.